shared/common-adapters/icon.stories.tsx

```diff
@@ -23,8 +23,8 @@ const commonProps = {
 
 const load = () => {
   const sizes = {}
-  Object.keys(iconMeta).map((type: IconType) => {
-    const meta = iconMeta[type]
+  Object.keys(iconMeta).map((type: any) => {
```

Comment:
what's going on here?

Reply:
TS wants the cast to make sense so it wants conflicting types (aka `string` => `IconType`) to make sense, and a way to do the dangerous cast (which we're doing here, cause its a story and its not that scary) is `string => any => IconType` (below). It could be `unknown` or even better IconMeta could be typed better